test/evergreen.yml

```diff
@@ -5862,7 +5863,6 @@ buildvariants:
   - ubuntu2004-small
```

Comment:
Up on line [5824](https://github.com/wiredtiger/wiredtiger/pull/10464/files#diff-42f20e21217bd7d4b14489056e1ea24ea2b87599c85413d49182a3b0b721c132R5824) we're running `.stress-test-sanitizer` on a variant with tcmalloc enabled. I commented out these tests in my proof of concept patch.

Reply:
I've now disabled tcmalloc, as I saw a couple of tcmalloc related failures in the *everything* patch build.

Reply:
I think with @mickgraham-mongodb help this is now addressed. I'll be pushing again.